core/src/test/groovy/jenkins/security/RSAConfidentialKeyTest.groovy

```diff
@@ -0,0 +1,25 @@
+package jenkins.security
+
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ *
+ *
+ * @author Kohsuke Kawaguchi
+ */
+class RSAConfidentialKeyTest {
+    @Rule
```

Comment:
Don't worry @jglick as usual will convert to java and commit with some other changes :)